gnucash/gnome-utils/gnc-main-window.c

```diff
@@ -312,6 +313,11 @@ static GtkActionEntry gnc_menu_actions [] =
         N_("Quit this application"),
         G_CALLBACK (gnc_main_window_cmd_file_quit)
     },
+    {
+        "DevLogAction", NULL, N_("_Developer Log"), NULL,
+        N_("Show developer log"),
```

Comment:
String Freeze!

Reply:
Relax, @fellen, this is still preliminary. It won't be done in time for 4.6.